contracts/dev-utils/contracts/src/DevUtils.sol

```diff
@@ -26,14 +26,12 @@ import "@0x/contracts-utils/contracts/src/LibEIP712.sol";
 import "@0x/contracts-utils/contracts/src/LibBytes.sol";
 import "./OrderValidationUtils.sol";
 import "./OrderTransferSimulationUtils.sol";
-import "./LibTransactionDecoder.sol";
 import "./EthBalanceChecker.sol";
 
 
 // solhint-disable no-empty-blocks
 contract DevUtils is
     OrderValidationUtils,
-    LibTransactionDecoder,
```

Comment:
Would it make more sense to turn this into a public library and expose it through `DevUtils`?

Reply:
I thought about that. In short, I'm not sure what would make more sense. I'm happy to do that.

Reply:
Just seems like this approach could get untenable having to juggle so many addresses. But I don't even know if our deployment tools are up to the task of external linkage. Maybe just punt on it for now and I'll play with it when I'm doing the dydx validation.

Reply:
I agree that we should probably make this a public library. I'm going to punt on this for this PR so that we can get this fix to TokenTrove, but feel free to put this on my plate when it comes up again.